modules/relocation-of-openshift-jenkins-images.adoc

```diff
@@ -0,0 +1,63 @@
+:_content-type: CONCEPT
+
+[id="relocation-of-openshift-jenkins-images_{context}"]
+= Relocation of OpenShift Jenkins images
+
+== What stays the same with the OpenShift Jenkins Image?
+* The Cluster Samples Operator manages the `ImageStream` object and pod templates for operating the OpenShift Jenkins images.
+* By default, the Jenkins `DeploymentConfig` object from the Jenkins pod template triggers a redeployment when the Jenkins image changes. By default, this image is referenced by the `jenkins:2` image stream tag of Jenkins image stream in the `openshift` namespace in the `ImageStream` YAML file in the Samples Operator payload.
+* If you upgrade from {product-title} 4.10 and earlier to 4.11, the deprecated `maven` and `nodejs` pod templates are still in the default image configuration.
+* If you upgrade from {product-title} 4.10 and earlier to 4.11, the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams still exist in your cluster. To maintain these image streams, see the “So what happens with the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams in the `openshift` namespace?” section below.
+
+== What changes in the support matrix of the OpenShift Jenkins image?
+
+Each new image in the `ocp-tools-4` repository in the `registry.redhat.io` registry supports multiple versions of {product-title}. When Red Hat updates one of these new images, it is available for all versions at the same time. This is ideal, for example, when Red Hat updates an image in response to a security advisory.
```

Comment:
@coreydaley @akram - as I recall in the early days of 4.11 dev, @adambkaplan mentioned there would be some sort of docs related to the CPaas image that would details what versions of OCP would be supported by given versions of the CPaas images (where between you and and @jitendar-singh someone validated against those various versions of OCP).

Ideally, once you all created that document, @rolfedh could link to it here.

Reply:
Any time is a good time for feedback, @gabemontero. I'll follow up on this with the team.

Reply:
@gabemontero that's on my TODO list....but, as I am having hard times in stabilizing the images released by CPaaS I am postponing this a little little.
But, it is in my plate.